### PR TITLE
Np 13 search functionality

### PR DESCRIPTION
**Pull request by DeimantasLapiniauskas**

### CODE REVIEW

File: src/components/helpers/get.js
Comment (FlorijanDem) [inline src/components/helpers/get.js:1]: Deprecated, use CRUD

**Comment by FlorijanDem** [comment]
Search in movies section have warning while search 
image

**Comment by FlorijanDem** [comment]
In this pull request you have deprecated components like "src/components/Movies/Movie.jsx", need some clean ups.

**Review by 13urn0ut — APPROVED** [review]
looks good to me

**Pull request merged**